pyresample/spherical.py

```diff
@@ -366,19 +366,19 @@ def __init__(self, start, end):
     def __eq__(self, other):
         """Check equality."""
         if self.start == other.start and self.end == other.end:
```

Comment:
Github is being annoying and not letting me select this full method

```suggestion
        return self.start == other.start and self.end == other.end
```